src/__tests__/unit/util/args.test.ts

```diff
@@ -1,20 +1,36 @@
+const processRunningPath = process.cwd();
+
 jest.mock('ts-command-line-args', () => ({
   __esModule: true,
   parse: () => {
     switch (process.env.result) {
+      case 'throw-error-object':
+        throw new CliInputError(MANIFEST_IS_MISSING);
       case 'error':
+        throw MANIFEST_IS_MISSING;
```

Comment:
do we need to support these two cases?
both are throwing error, one with error class, the other without

Reply:
yeah, because of `validateAndParseProcessArgs` function:
```
const validateAndParseProcessArgs = () => {
  try {
    return parse<ManifestProcessArgs>(ARGS);
  } catch (error) {
    if (error instanceof Error) {
      throw new CliInputError(error.message);
    }

    throw error;
  }
};
```

Reply:
ok you're covering here `throw error;` case